scripts/test-e2e-local.js

```diff
@@ -37,6 +32,15 @@ const {
   expandHermesSourceTarball,
 } = require('react-native/scripts/hermes/hermes-utils.js');
 
+const {
+  checkPackagerRunning,
+  maybeLaunchAndroidEmulator,
+  isPackagerRunning,
+  launchPackagerInSeparateWindow,
+  setupCircleCIArtifacts,
+  prepareArtifacts,
+} = require('./testing-utils');
```

Comment:
headsup: a few of these imports are completely unused: `generateAndroidArtifacts`, `generateiOSArtifacts`, `downloadHermesSourceTarball`, `expandHermesSourceTarball`, `isPackagerRunning`, `cp`.